armi/reactor/converters/uniformMesh.py

```diff
@@ -85,6 +86,317 @@ def converterFactory(globalFluxOptions):
         )
 
 
+class UniformMeshGenerator:
+    """
+    The :py:class:`UniformMeshGenerator <armi.reactor.converters.UniformMeshGenerator>`
+    generates a common axial mesh to for the uniform mesh converter to use. The generation
+    algorithm starts with the simple ``average1DWithinTolerance`` utility function to compute
+    a representative "average" of the assembly meshes in the reactor. It then modifies that
+    mesh to more faithfully represent important material boundaries of fuel and control absorber
+    material.
+
+    The decusping feature is controlled with the case setting ``uniformMeshMinimumSize``. If no
+    value is provided for this setting, the uniform mesh generator will skip the decusping step
+    and just provide the result of ``_computeAverageAxialMesh``.
+    """
+
+    FLOAT_ROUNDING_DECIMALS = 12
```

Comment:
maybe bring this down to 10. i am nervous that since it is not configurable, if you had bigger numbers  (>10kl) it would error out. either that or add comment for basis of 12 places

Reply:
I was initially rounding to 5 decimal places because I thought anything less than that is just insignificant for meshing, and that would reduce the amount of redundant mesh points that need to be filtered out.

But I ran into a problem because the `b.p.zbottom` and `b.p.ztop` are not rounded. So if, for example, `fuelBlock.p.ztop = 100.1234567`, you'd get a mesh point at `100.12345`, and then 0.0000067 cm of that fuel would get blended into the next block above it. I went to 12 to avoid this rounding problem. But rounding to 12 probably isn't serving any purpose. I will test without the rounding and see if the functionality is the same.